themes/bootstrap3/templates/record/view.phtml

```diff
@@ -14,44 +14,58 @@
 
   // Set up breadcrumbs:
   $this->layout()->breadcrumbs = '<li>' . $this->searchMemory()->getLastSearchLink($this->transEsc('Search'), '', '</li> ') .
-    '<li class="active">' . $this->recordLink()->getBreadcrumb($this->driver) . '</li> ';
+    '<li class="active" aria-current="page">' . $this->recordLink()->getBreadcrumb($this->driver) . '</li> ';
   $this->layout()->title = $this->driver->getShortTitle();
 ?>
 
 <?php if (isset($this->scrollData) && ($this->scrollData['previousRecord'] || $this->scrollData['nextRecord'])): ?>
   <ul class="pager hidden-print">
     <?php if ($this->scrollData['previousRecord']): ?>
       <?php if ($this->scrollData['firstRecord']): ?>
-        <li>
-          <a href="<?=$this->recordLink()->getUrl($this->scrollData['firstRecord'])?>" title="<?=$this->transEsc('First Search Result')?>" rel="nofollow"><span aria-hidden="true">&laquo;</span> <?=$this->transEsc('First')?></a>
+        <li role="none">
+          <a href="<?=$this->recordLink()->getUrl($this->scrollData['firstRecord'])?>" title="<?=$this->transEsc('First Search Result')?>" aria-label="<?=$this->transEsc('First Search Result')?>" rel="nofollow">
```

Comment:
The language file is missing translations for "First Search Result," "Previous Search Result," "Next Search Result" and "Last Search Result." This appears to be a pre-existing bug, but we should address it now that we've noticed it.

Reply:
I'm also wondering if this is overly verbose, with title and aria-label set to the same thing, and with both of those values being a longer form of the text found inside the link. Did you test how this behaves in a real-world screen reader?

Reply:
[You're right that this will be redundant to screen readers](https://www.deque.com/blog/text-links-practices-screen-readers/).

The title is useful for desktop mouse users. One option for them is to use a tooltip library that utilizes aria-labels and roles, [like this one](https://github.com/ghosh/microtip#usage). That would give the same visual feedback and encourage proper markup.

Reply:
I like the idea of using microtip to sort this out. Do you think that's worth addressing here, or something we should consider as a separate phase of work?